# Robot name goes here
car_controller:
  # default controller
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 10

  # Controller 1
  rear_controller:
    type: velocity_controllers/JointVelocityController
    joint: rear_joint
    # pid : {p: 100.0, i: 0.1, d: 0.1}

  # Controller 2
  front_left_steering:
    type: position_controllers/JointPositionController
    joint: left_joint
    # pid : {p: 100.0, i: 0.1, d: 0.1}  
  
  # Controller 3
  front_right_steering:
    type: position_controllers/JointPositionController
    joint: right_joint
    # pid : {p: 100.0, i: 0.1, d: 0.1}  

  
  gazebo_ros_control:
    pid_gains:
      rear_joint: {p: 10.0, i: 0.001, d: 0.001}
      right_joint: {p: 100.0, i: 0.001, d: 2.0}
      left_joint: {p: 100.0, i: 0.001, d: 2.0}
